security/ClientDirectedCertificateTransfer.txt

```diff
@@ -0,0 +1,24 @@
+/*
+@startuml
+autonumber
+
+title Client-directed Certificate Transfer
+
+participant "Credential Management Service" as CMS
+participant "Device" as DEV
+
+note over CMS, DEV
+The Ownership Credential should be used to establish a secure connection.
+end note
+
+CMS->DEV: RETRIEVE /oic/sec/csr
+DEV->CMS: RSP [{"csr":"<contents of CSR>", "encoding":"<supported-encoding-type>"}]
+
+CMS->DEV: UPDATE /oic/sec/cred [{"credtype":8, "subject":"deviceuuid", "credusage":"primary_cert", "publicdata":"<DER-encoded device and CA certificate chain in base64>"}]
```

Comment:
Break this line also into two lines
CMS->DEV: UPDATE /oic/sec/cred [{"credtype":8, "subject":"deviceuuid", "credusage":"primary_cert", \n"publicdata":"<DER-encoded device and CA certificate chain in base64>"}]